app/components/syntax-highlighted-code.ts

```diff
@@ -23,18 +23,13 @@ export default class SyntaxHighlightedCodeComponent extends Component<Signature>
     super(owner, args);
 
     const highlighterPromise = getOrCreateCachedHighlighterPromise(`${this.args.theme}-${this.args.language}`, {
-      theme: this.args.theme,
-      // @ts-expect-error - shiki types are not up to date
+      themes: [this.args.theme],
       langs: [this.args.language],
     });
 
-    const lineOptions = (this.args.highlightedLines || '')
```

Comment:
@libmartinito Why was this removed? Did you check to see where it is used?

Reply:
line options were deprecated in favor of [transforms](https://shiki.matsu.io/packages/transformers#transformers). highlighted lines were passed to the syntax highlighted code component only from code walkthroughs. iirc, we don't use code walkthroughs any more.

Reply:
Ah, gotcha - We don't create new code walkthroughs, but we do have them linked in multiple places, so we'd want the highlights to stay intact. From plausible: 

<img width="811" alt="Screenshot 2024-04-26 at 10 40 59" src="https://github.com/codecrafters-io/frontend/assets/3893573/6640c3f3-c3c5-4b45-a903-c0a96b03c771">

And in general it's a feature we've worked on and will find useful in the future, wouldn't want to discard